scripts/redshift/transformers/specimen_library.py

```diff
@@ -119,50 +119,58 @@ def create_ontology_resolver(self, *metadata_dict_lists):
 
         return resolver
 
-    def parse_donor_json(self, bundle_path):
+    def parse_specimen_json(self, bundle_path):
+        specimen_json_paths = glob.glob(
+            os.path.join(bundle_path, "specimen_from_organism_*.json"))
         donor_json_paths = glob.glob(
             os.path.join(bundle_path, "donor_organism_*.json"))
 
         genera = set()
         ethnicities = set()
         diseases = set()
         dev_stages = set()
-        donorkeys = []
+        specimenkeys = []
 
-        for donor_json_path in donor_json_paths:
-            donor_dict = json.load(open(donor_json_path))
-            key = donor_dict["provenance"]["document_id"]
+        for specimen_json_path in specimen_json_paths:
+            specimen_dict = json.load(open(specimen_json_path))
+            key = specimen_dict["provenance"]["document_id"]
+            specimenkeys.append(key)
 
-            genus_species_list = donor_dict.get("genus_species", [])
+            genus_species_list = specimen_dict.get("genus_species", [])
             if genus_species_list:
                 genus_species = genus_species_list[0].get("ontology", "").upper()
             else:
                 genus_species = ""
+            genera.add(genus_species)
+
+            disease_list = specimen_dict.get("diseases", [])
+            if disease_list:
+                disease = disease_list[0].get("ontology").upper()
+            else:
+                disease = ""
+
+            diseases.add(disease)
+
+        for donor_json_path in donor_json_paths:
+            donor_dict = json.load(open(donor_json_path))
+
 
             ethnicity_list = donor_dict.get("human_specific", {}).get("ethnicity", [])
             if ethnicity_list:
                 ethnicity = ethnicity_list[0].get("ontology").upper()
             else:
                 ethnicity = ""
 
-            disease_list = donor_dict.get("diseases", [])
-            if disease_list:
-                disease = disease_list[0].get("ontology").upper()
-            else:
-                disease = ""
 
             dev_stage = donor_dict.get("development_stage", {}).get("ontology", "").upper()
 
-            donorkeys.append(key)
-            genera.add(genus_species)
             ethnicities.add(ethnicity)
-            diseases.add(disease)
             dev_stages.add(dev_stage)
 
-        # Some cells have multiple donors. Only record the donor information if
+        # Some cells have multiple donors/specimens. Only record the donor information if
```

Comment:
Is there value in/do we plan on supporting multi-valued fields in the future? ex. list of specimen keys in `cell` table

Reply:
I think there is, but I don't know how to implement it.

Reply:
Sounds good. We could look into it further, but a `LIKE` query on `donorkey` where the value is a CSV may work

Reply:
Yeah we'd have to see how that impacts performance. They tell you not to use `LIKE` very much in the [redshift docs](https://docs.aws.amazon.com/redshift/latest/dg/c_designing-queries-best-practices.html).